test/RibbonThetaVault.ts

```diff
@@ -437,18 +472,20 @@ function behavesLikeRibbonOptionsVault(params: {
       feeRecipient = feeRecipientSigner.address;
 
       const TestVolOracle = await getContractFactory(
-        TestVolOracle_ABI,
-        TestVolOracle_BYTECODE,
-        ownerSigner
+        ManualVolOracle_ABI,
+        ManualVolOracle_BYTECODE,
+        keeperSigner
       );
 
-      volOracle = await TestVolOracle.deploy(PERIOD, 7);
+      volOracle = await TestVolOracle.deploy(keeper);
 
-      await volOracle.initPool(
-        asset === WETH_ADDRESS[chainId]
-          ? ETH_USDC_POOL[chainId]
-          : WBTC_USDC_POOL[chainId]
-      );
+      await volOracle.setAnnualizedVol(ETH_USDC_POOL[chainId], 106480000);
+      await volOracle.setAnnualizedVol(WBTC_USDC_POOL[chainId], 106480000);
+
+      for (let i = 0; i < 4; i++) {
+        const topOfPeriod = (await time.getTopOfPeriod()) + time.PERIOD;
+        await time.increaseTo(topOfPeriod);
+      }
```

Comment:
Trying to understand this... why do we have to increase the time to top of period 4 times?

Reply:
Fixed, I had to increase the option's expiry date to the future week.